features/admin/dashboard.feature

```diff
@@ -0,0 +1,41 @@
+@admin_dashboard
+Feature: Statistics dashboard
+    In order to have an overview of my sales
+    As an Administrator
+    I want to see overall statistics on my admin dashboard
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And the store has a product "Sylius T-Shirt"
+        And this product has "Red XL" variant priced at "€40"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Seeing basic statistics for entire store
+        Given 3 customers have placed 4 orders for total of "€8566.00"
+        And then 2 more customers have placed 2 orders for total of "€459.00"
+        When I open administration dashboard
+        Then I should see 6 new orders
+        And I should see 6 new customers
```

Comment:
3 customers + 2 customers != 6 customers (I know there's also an admin, but it seems werid)